### DISCUSSION TITLE
Reading OSI-SAF data into METplus

### DISCUSSION THREAD

**Discussion by GwenChen-NOAA** [dtcenter/METplus#1836] [File I/O] [answered]
Dear METplus team,

I tried to use the OSI-SAF data to validate sea ice concentration with METplus.  I used plot_data_plane to test read the data but encountered the following error:

ERROR  : get_nc_var_att_double() -> Cannot get "longitude_of_projection_origin" from Polar_Stereographic_Grid variable.

The OSI-SAF data are in polar stereographic grid.  Can METplus directly read in this dataset?  I uploaded two sample data files to the ftp site under chen_data.  Thanks in advance for your help!

**Comment by JohnHalleyGotway** [comment, marked as answer]
@GwenChen-NOAA FYI this is the same issue as we discussed with the NSIDC Sea Ice Edge files in dtcenter/METplus#1718.

The changes described in issue dtcenter/MET#2218 will also fix the issue you're experiences with this OSI-SAF data.

@hsoh-u is working on a fix for this but we're having some internal discussions about how to handle the `standard_parallel` setting.

Once that issue is fixed for both the develop and main_v10.1 branches, we can create a MET version 10.1.3 bugfix release.

**Reply by GwenChen-NOAA** [reply]
Thanks for the info!  We will follow the [MET#2218](https://github.com/dtcenter/MET/issues/2218) for both NSIDC and OSI-SAF data.

**Reply by JohnHalleyGotway** [reply]
Great, I'll go ahead and mark this discussion as being answered. Please feel free to comment on MET#2218 as needed.

In addition, we'll be reaching out to you in the next few weeks to test out the changes for the MET 10.1.3 bugfix release on Hera to make sure that it'll fix these issues well.
